core/src/main/java/org/eclipse/hono/util/Constants.java

```diff
@@ -57,6 +57,12 @@
      */
     public static final String PROTOCOL_ADAPTER_TYPE_AMQP= "hono-amqp";
 
+    /**
+     * Indicates that the AMQP adapter cannot process the request due to a client error. A bad inbound
```

Comment:
if this constant is intended to be used by the AMQP adapter only, then why don't we define the constant there? Or is this constant intended to be used by any type of server exposing an AMQP endpoint instead?

Reply:
ok, sure

Reply:
If it is not intended to be used by other components then move it to the AMQP adapter, otherwise adapt the JavaDoc.